### DISCUSSION TITLE
Question: What is performance of data-importer normally?

### DISCUSSION THREAD

**Discussion by ankit-yy** [pimcore/pimcore#9960] [Q&A / Support] [answered]
How many records can be imported in a minute using data-importer (sequentially or parallely)?
https://github.com/pimcore/data-importer

let's say on 8GB ram and sufficient space.
i just want a benchmark so i can compare with my setup.

Thanks

**Comment by fashxp** [comment, marked as answer]
This depends on many factors ... like
- complexity of data model
- mapping configuration and transformation pipelines
- database configuration and power
- cpu power
- size of database
- other jobs the system has to process
- etc. 
We don't have any generic benchmarks yet. 

Under the hood, the data importer just uses the Pimcore php API ...
